x-pack/plugins/security_solution/public/helpers.tsx

```diff
@@ -176,7 +176,7 @@ export const isDetectionsPath = (pathname: string): boolean => {
 
 export const isDashboardViewPath = (pathname: string): boolean =>
   matchPath(pathname, {
-    path: `/${DASHBOARDS_PATH}/:id`,
+    path: `${DASHBOARDS_PATH}/:detailName`,
```

Comment:
We don't need `/` here as `DASHBOARDS_PATH = /dashboards` (x-pack/plugins/security_solution/common/constants.ts line 82)